ballerina/Module.md

```diff
@@ -18,7 +18,3 @@ import ballerina/sql;
 import ballerinax/mssql;
 import ballerinax/mssql.driver as _;
 ```
-
-# Useful Links
-* Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
```

Comment:
Why is this removed?

Reply:
We usually don't have the Useful links section in Module.md. Instead we have them in Package.md

e.g.
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Module.md
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Package.md